src/js/components/Chart/doc.js

```diff
@@ -80,6 +80,18 @@ export const doc = Chart => {
       Related to 'overflow', this allows control over how much space
       is available for bars and points to overflow into.`,
     ),
+    point: PropTypes.oneOf([
+      'circle',
+      'diamond',
+      'square',
+      'star',
+      'triangle',
+      'triangleDown',
+    ]).description(
+      `When using a 'point' type, what shape the points should use.
```

Comment:
```suggestion
      `When using a 'point' type and not using 'round' prop, what shape the points should use.
```
Just an idea of how to communicate the dependency on the `round` prop (see my other comment in this file)...

Reply:
Actually, it's the other way around. If `point` isn't specified, then `round` affects it, to align with current behavior.